code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -218,8 +218,12 @@
 
 	if(fire_stacks > 0)
 		msg += "[t_He] [t_is] covered in something flammable.\n"
-	if(fire_stacks < 0)
+	if(fire_stacks < 0 && fire_stacks > -1)
```

Comment:
this statement looks funny and doesnt seem like it would work

you could use a switch like
```
switch(fire_stacks)
  
  if(1 to INFINITY)
    flammable something
  if(-1)
     wet etc
  if(-2 to -4)
    etc
  if(-4 to -INFINITY)
    etc
```

Reply:
i think i did test it and it worked as expected, but this is infinitely cleaner and easier to read/understand. I always forget switch() is a thing